week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -69,17 +89,21 @@ function shakeBall() {
       || answer === "Better not tell you now."
       || answer === "Cannot predict now."
       || answer === "Concentrate and ask again.") {
-      return "Negative"
+      return "negative"
     } else if (answer === "Don't count on it."
       || answer === "My reply is no."
       || answer === "My sources say no."
       || answer === "Outlook not so good."
       || answer === "Very doubtful.") {
-      return "Very negative"
+      return "very negative"
     }
-    return answer
   }
 }
+function shakeBall() {
+  console.log("The ball has shaken!");
+  return "My reply is no."
```

Comment:
This is generally looking great, but there's just one thing left - the magic 8 ball always returns the same thing! Let's try to always make it return a different thing - you already know how to get an element from an array, and we've seen that you can use `Math.random` to generate a random number, so how can we put the two together to get a random answer from your `ball` array?